src/Android/Xamarin.Android/Samples/MapView/AnimateImageOverlay/readme.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "category": "MapView",
+    "description": "Animate a series of images with an image overlay.",
+    "formal_name": "AnimateImageOverlay",
+    "ignore": false,
+    "images": [
+        "AnimateImageOverlay.jpg"
+    ],
+    "keywords": [
+        "3d",
+        "animation",
+        "drone",
+        "dynamic",
+        "image frame",
+        "image overlay",
+        "real time",
+        "rendering"
+    ],
+    "nuget_packages": {
+        "Esri.ArcGISRuntime": "100.8.0"
+    },
+    "offline_data": [],
+    "redirect_from": [
+        "/net/latest/android/sample-code/animateimageoverlay.htm"
+    ],
+    "relevant_apis": [
+        "ImageFrame",
+        "ImageOverlay",
+        "SceneView"
+    ],
+    "snippets": [
+        "AnimateImageOverlay.cs"
```

Comment:
Quick note about the layout- Because we no longer support VS 2017 for Xamarin platforms, we can now use .xml instead of .axml for android layouts. Eventually I will rename all of the other layouts accordingly to match the VS 2019 format.